controllers/gslb_controller.go

```diff
@@ -240,6 +243,23 @@ func (r *GslbReconciler) SetupWithManager(mgr ctrl.Manager) error {
 			},
 		}
 
+		strToInt := func(string string) int {
```

Comment:
For better readability, consider to put function declaration to the beginning of the caller function (between variable, constant declarations).

Reply:
@kuritka could you point me to the location you mean? I'm not 100% sure I'm getting you right

Reply:
@somaritane renamed to `str`, amedned)

Reply:
@ytsarev basically is up to you. You can keep it as it is, because the function is preceded only by some initializations. Also strToInt is used only in context of createGslbFromIngress handler and nowhere else.

Another place to put this func is under SetupWithManager, so it is at the same level as other funcs.

Both seems totally equal.

Reply:
>  Also strToInt is used only in context of createGslbFromIngress handler and nowhere else.

That's exactly why I decided to declare it inline and next to the actual invocation.
I think I will keep it as it is for now and we move it out to some other level when/if there will be another case of `strToInt` usage in the codebase.

Thanks a lot for a careful review.